modules/ossm-config-dist-trac.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/customizing-installation-ossm.adoc
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+[id="ossm-config-dist-trac_{context}"]
+= Distributed tracing
+
+Distributed Tracing is the process of tracking the performance of individual services in an application by tracing the path of the service calls in the application. Each time a user takes action in an application, a request is executed that might require many services to interact to produce a response. The path of this request is called a distributed transaction.
+
+{ProductName} uses Jaeger to allow developers to visualize call flows in a microservice application. 
```

Comment:
allow

Reply:
IBM SG: Second person
As much as possible, use the second person, which is indicated by the personal pronoun you. The second person focuses on the reader. For example:
"When you create a database, you must provide a unique name."
Here, you could rephrase this to something more like:
"As a <user's role>, you can use <name of feature or capability> to <synopsis of user story>."